mkdocs.yml

```diff
@@ -33,6 +33,7 @@ nav:
       - Networking (CNI):                 networking.md
       - Runtime (CRI):                    runtime.md
       - Storage (CSI):                    storage.md
+      - System monitoring:                system-monitoring.md
```

Comment:
I'd still change the navigation title to "System Components Monitoring" even though it's a bit long. Otherwise, users might expect more than what this page offers.